shared/notifications/index.js

```diff
@@ -82,6 +82,10 @@ export const UI_NOTIFICATIONS = {
     id: 16,
     date: null,
   },
+  17: {
+    id: 17,
+    date: null, // ???
```

Comment:
We can probably remove the comment

Reply:
removed [here](https://github.com/MetaMask/metamask-extension/pull/16783/commits/50e7a508127bca166e362cccebff65518d447603)